ADQL.tex

```diff
@@ -2410,14 +2215,9 @@ \subsubsection{POLYGON}
 numeric values.
 For example:
 \begin{verbatim}
-    POLYGON(
-        t1.ra,
-        t1.dec + 5,
-        t1.ra  - 5,
-        t1.dec - 5,
-        t1.ra  - 5,
-        t1.dec + 5,
-        )
+    POLYGON(t1.ra    , t1.dec + 5,
+            t1.ra - 5, t1.dec - 5,
+            t1.ra - 5, t1.dec + 5,)
```

Comment:
Remove the filnal trailing comma here (error inherited from previous version).

Reply:
Oops. I already fixed one like that, but I have not seen this one. Now fixed. Thanks.